src/cityreader/cityreader.py

```diff
@@ -59,13 +78,25 @@ def cityreader(cities=[]):
 # Salt Lake City: (40.7774,-111.9301)
 
 # TODO Get latitude and longitude values from the user
+pairOne = input('Input first coordinates: lat,lon ')
```

Comment:
good attempt overall here, and I'm happy that you put in the effort to at least log what the user is doing!